app/code/Transoft/Blog/Controller/Adminhtml/Post/Edit.php

```diff
@@ -0,0 +1,45 @@
+<?php
+declare(strict_types=1);
+namespace Transoft\Blog\Controller\Adminhtml\Post;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Magento\Framework\View\Result\PageFactory;
+
+/**
+ * Class Edit
+ *
+ * Edit controller class
+ */
+class Edit extends Action implements HttpGetActionInterface
+{
+    const ADMIN_RESOURCE = 'Post';
```

Comment:
This looks weird. Should **Transoft_Blog::post** or other resource. Isn't it?

Reply:
fixed